SPECS/libdwarf/libdwarf.spec

```diff
@@ -1,15 +1,16 @@
 Name:          libdwarf
-Version:       20200114
-Release:       3%{?dist}
-Summary:       Library to access the DWARF Debugging file format 
+Version:       0.9.0
```

Comment:
Please also remove the `SPECS-EXTENDED/libdwarf`.

Reply:
ah it seems there was a mistake in my cherry-pick. I fixed it. Thanks for catching it!